### DISCUSSION TITLE
Turn off mypy and pycodestyle

### DISCUSSION THREAD

**Discussion by txstc55** [zed-industries/zed#21015] [Feature Requests]
After the new update, all the warning and errors from mypy and pycodestyle are shown on screen. How can I disable that? I already tried to put the pyproject.toml under the root folder and the package folder with:
[tool.mypy]
files = []
to disable them, but didnt work.
I'm not using a venv for python, or rather, I'm using the root python as my vitual environment shown in zed

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
